binplace.targets

```diff
@@ -10,21 +10,35 @@
     <BinPlaceStuffDependsOn Condition="'$(IsReferenceAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceReferenceAssembly</BinPlaceStuffDependsOn>
     <BinPlaceStuffDependsOn Condition="'$(IsRuntimeAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceRuntimeAssembly</BinPlaceStuffDependsOn>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <BinplaceConfiguration Include="$(BuildConfiguration)">
+      <RefPath>$(RefPath)</RefPath>
+      <RuntimePath>$(RuntimePath)</RuntimePath>
+    </BinplaceConfiguration>
+    <BinplaceConfiguration Condition="'$(IsNETCoreApp)' == 'true'" Include="netcoreapp-$(OSGroup)">
```

Comment:
ArchGroup isn't persisted in the project configuration string https://github.com/dotnet/corefx/blob/master/src/Tools/GenerateProps/properties.props#L32